### PR TITLE
landing page styling & cleanup of API folder merge conflicts

### PR DESCRIPTION
**Pull request by serpient**

### CODE REVIEW

**Comment by serpient** [comment]
All those conflicts in the Login.tsx finally showed up as merge conflicts when i tried to push my work in. I resolved the merge conflicts to match my current Login.tsx file (which you worked on). Take a look and merge the pull request if its ok.

File: README.md [hunk 1]
```diff
@@ -1,27 +1,40 @@
...
+git remote add upstream https://github.com/chingu-voyage4/Bears-Team-11.git
+npm install -g create-react-app
+npm install
+cd bears-team-11/
+npm start
+```
 ## Running the Server
-`npm run start-server` runs the express server on port `8080`.
+- `npm run start-server` runs the express server on port `8080`.
+- Open Postman app or use your browser to reach `http://localhost:8080/`
```
Comment (luoto) [inline README.md]: Lets provide a link to Postman for convenience: [Postman](https://www.getpostman.com/).

File: README.md [hunk 2]
```diff
@@ -1,27 +1,40 @@
...
-- Home:  `/home`. ( only if already loged in otherwise redirects to `/`).
-- Login: `/login?email=ram@hello.com&password=123`.
-- Signup: `/signup?email=hello@hello.com&password=123&username=hello&firstName=yourFirstName&lastName=yourLastName`.
-- SignOut: `/signout`.
-
-Currently using [nodemon](https://github.com/remy/nodemon) to watch and restart server for changes. We are also currently not using babel for trancompilation for the servside code.
+- Home:  `/home` ( only if already loged in otherwise redirects to `/`).
+- Login: `/login` (takes email and password params)
+- Signup: `/signup` (takes first name, last name, username, email, and password params)
+- SignOut: `/signout`
```
Comment (luoto) [inline README.md]: This should work for now, but as this grow, we should think about putting this into a table format like in our gDocs.

File: src/Projects.tsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ class Project extends React.Component<Props, State> {
     var data = this.props.project;
     var roles;
     if (data.lookingFor && data.lookingFor.length > 1) {
-      roles = data.lookingFor[0] + ', ' + data.lookingFor[1];
+      roles = data.lookingFor[0] + ',  ' + data.lookingFor[1];
```
Comment (luoto) [inline src/Projects.tsx]: Is this intentional? Looks like the original version should be fine. Styling should be done with css.

File: api/passport/init.js [hunk 4]
```diff
@@ -1,4 +1,3 @@
-<<<<<<< HEAD
 /*
  This is the initialization file for passport
```
Comment (ilvcs) [inline api/passport/init.js]: this is good.

File: README.md (hunk 2 above)
Comment (ilvcs) [reply README.md]: we have to add like `/api/home`

**Pull request merged**